StaticCache.php

```diff
@@ -0,0 +1,38 @@
+<?
+require_once '../Matryoshka/library/iFixit/Matryoshka/Backend.php';
```

Comment:
I would like to do that, although I'm not sure how a clean interface would be made other than to have an interface that they should implement.

The other option would just have two simple callbacks that are injected to the TokenBucket that it would use for get and set. The benefits of this would be that it would be more simple, and probably less friction.